com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/cf/metadata/util/MtaMetadataUtil.java

```diff
@@ -44,6 +45,10 @@ public static boolean hasMtaMetadata(CloudEntity entity) {
                      .anyMatch(MTA_METADATA_LABELS::contains);
     }
 
+    public static String getHashedMtaId(String mtaId) {
+        return DigestUtils.md5DigestAsHex(mtaId.getBytes());
```

Comment:
How do we guarantee hash size?

Reply:
Guaranteed by the hashing algorithm itself. MD5 = 128 bits = 16 bytes = 32 hex digits